EIPS/eip-7530.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 7530
+title: EVM Profiles for ZK Rollups
```

Comment:
```suggestion
title: EVM Profiles for ZK Rollups based on shanghai hardfork
```
as well as description, since the EVM profiles described are  specific and concrete w.r.t. the hardfork